game.py

```diff
@@ -20,8 +20,35 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
-
+    # Initialize the correct_letter_guess_statuses dictionary
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+
+    # Loop until the player wins or loses
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
+        print("\nWord: " + generate_word_progress_string(snowman_word, correct_letter_guess_statuses))
+        print("Wrong guesses:", " ".join(sorted(wrong_guesses_list)))
+        # Get a letter from the user
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        # Check if the user input is in the word
+        if user_input in correct_letter_guess_statuses:
+            print("This letter is in the word!")
```

Comment:
Just a small thing, but when an incorrect letter is guessed, it tells you which letter was guessed. It might be a good idea to keep things consistent by doing the same thing if a correct letter is guessed!

```suggestion
            print(f"The letter {user_input} is in the word!")
```